examples/morpheusvm/externalsubscriber/sidecar.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package externalsubscriber
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"go.uber.org/zap"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/genesis"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/rpc"
+	"github.com/ava-labs/hypersdk/extension/indexer"
+	"github.com/ava-labs/hypersdk/fees"
+	"github.com/ava-labs/hypersdk/pebble"
+
+	ametrics "github.com/ava-labs/avalanchego/api/metrics"
+	mpb "github.com/ava-labs/hypersdk/examples/morpheusvm/proto"
+	pb "github.com/ava-labs/hypersdk/proto"
+	hstorage "github.com/ava-labs/hypersdk/storage"
+)
+
+// TODO: clean up directory structure
+var stdDBDir = "./stdDB/"
+
+type ExternalMorpheusAPI interface {
```

Comment:
We should make this a JSON RPC API https://github.com/ava-labs/hypersdk/pull/1252/#issuecomment-2274005695